packages/app/src/specs/generators/component/ReactComponentGeneratorStepOne.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div class="flex flex-col flex-grow justify-between">
+    <template v-if="generatedSpecError">
+      <EmptyGenerator
```

Comment:
Do we know if this flow works for CFR? Wouldn't the mutation need to know the `isDefault` for it to scaffold properly after then renaming?

Reply:
I haven't tried this yet, but I don't think it would need `isDefault` since it will just create an empty spec. It doesn't try to import anything

Reply:
Okay I tried this... it doesn't seem to work for either of the create from component flows, but I think it's for a different reason. Going to make an issue to look into this because we should have a fallback if something goes wrong.